packages/jest-runtime/src/index.ts

```diff
@@ -892,9 +892,17 @@ class Runtime {
       const createRequire = (modulePath: string | URL) => {
         const filename =
           typeof modulePath === 'string'
-            ? modulePath
+            ? modulePath.startsWith('file:///')
+              ? fileURLToPath(new URL(modulePath))
+              : modulePath
             : fileURLToPath(modulePath);
 
+        if (!path.isAbsolute(filename)) {
+          throw new TypeError(
```

Comment:
should set the `code` property to `ERR_INVALID_ARG_TYPE` as well 🙂

Reply:
@SimenB How can I set error code? Typescript does not allow me to do that for `Error` instance

Reply:
```ts
const error = new TypeError('blabla')
error.code = 'ERR_INVALID_ARG_TYPE';
throw error;
```

If TS doesn't allow you, just use a `@ts-ignore`